misc/config_tools/configurator/README.md

```diff
@@ -29,17 +34,17 @@ which provide `xmllint` command.
 ```bash
 sudo apt install git
 git clone https://github.com/projectacrn/acrn-hypervisor
-cd acrn-hypervisor/misc/config_tools/configurator
+cd acrn-hypervisor/misc/config_tools
+python3 -m pip install -r requirements.txt
```

Comment:
Why do we use this vs. a simple `pip3 install ...`?

Reply:
In some env, there has 2 or more versions of python, pip3 maybe is pip3.8, python3 maybe means python 3.6, in this time user use `pip3 install xxx` will install to python3.8's site-package, when user use `python3 yyy.script` will still got xxx module not found error.

More Infomation:  
1. [whats-the-difference-between-pip-install-and-python-m-pip-install](https://stackoverflow.com/questions/25749621/whats-the-difference-between-pip-install-and-python-m-pip-install)
2. [work-with-multiple-versions-of-python-installed-in-parallel](https://docs.python.org/3/installing/index.html#work-with-multiple-versions-of-python-installed-in-parallel)

Reply:
Thanks @Weiyi-Feng , I've learned something new today :-)